tests/testthat/test-geom-sf.R

```diff
@@ -99,18 +99,18 @@ test_that("geom_sf() removes rows containing missing aes", {
     colour = c("red", NA)
   )
 
-  p <- ggplot(pts) + geom_sf()
+  p <- ggplot(pts)
   expect_warning(
-    expect_identical(grob_xy_length(p + aes(size = size)), c(1L, 1L)),
+    expect_identical(grob_xy_length(p + geom_sf(aes(size = size))), c(1L, 1L)),
```

Comment:
I needed a tweak here because of this issue: https://github.com/tidyverse/ggplot2/issues/4960